### ISSUE TITLE
Migrate Registry to Quarkus 3 and to the Jakarta world

### ISSUE THREAD

**Issue by carlesarnal** [Apicurio/apicurio-registry#3180]

**Comment by jan-schumacher** [comment]
I don't understand why ist issue has just priority "normal". This is a major blocker when updating frameworks next to apicurio. Spring-Boot 3 is blocked by this as well.

**Comment by carlesarnal** [comment]
This is already done using Quarkus alpha versions for testing, Quarkus 3 final has not been even released.

**Comment by jan-schumacher** [comment]
It's the same in Spring-Boot 3/Spring Framework 6. Javax is gone there.

**Comment by carlesarnal** [comment]
I'm not sure what you mean. Since Apicurio Registry is a project depending on Quarkus (it's a Quarkus app), for us these dependencies are basically transitive dependencies and cannot be removed until Quarkus 3 (the version removing the javax stuff) is released. Once that is done (we already have a branch with the required changes, it's not merged because Quarkus 3 is still in beta), we will do a release with those changes including the Quarkus 3 migration and the javax stuff will be gone.

**Comment by EricWittmann** [comment]
Are there javax references in the Serdes classes or the registry client?  If yes then that could be problematic until we release a Quarkus 3 version.  If not, then it's not clear what integration is at risk.  Is someone packaging registry using Spring Boot and running it as a SB application?  I would be quite surprised if yes.
